database/controllers/dbm.js

```diff
@@ -48,8 +47,34 @@ module.exports = {
     //Description:  This function will return all User data from
     // the given start date to the given end date.  If start date
     // is undefined then all users will be returned.
-    getUser(from, to){
-        return true;
+    getUsers(from, to){
+        if(from === undefined){
```

Comment:
Am I using this wrong somehow? It is not returning any users. I used the registration page's get request route to trap and call the function. I printed both the return value and the result as follows:
![image](https://user-images.githubusercontent.com/33047755/35713571-e5350108-077c-11e8-97a3-fdd812e8ad74.png)

...and got the following output:
![image](https://user-images.githubusercontent.com/33047755/35713594-03e80ffa-077d-11e8-8d00-19879bfcf460.png)

Reply:
Nevermind, I figured it out. I noticed the credentials you checked in were different than the ones I had saved before from the "DataBase Setup and Connection Test" file. It connected successfully but to the wrong tables, I assume. The old tables don't support this new function. However, when I use the credentials you checked in, it returns the users as desired:
![image](https://user-images.githubusercontent.com/33047755/35714196-1aefafb6-0780-11e8-8947-668624fec56f.png)

Reply:
Yeah, sorry should’ve mentioned this PR uses different credentials then we’ve been using.  

I was planning to update the database setup and connection doc if/after this got merged to master.  I’ll also put in slack or in the doc itself the username and password so everyone can access the tables/data directly.

Reply:
Also is that the only result you got or was there more that just wasn’t captured?  I believe a simple getUsers call with no arguments should return more then one result.

Reply:
I got an array of users back; I just expanded one of them for the sake of brevity.

Reply:
Okay, cool.